.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install npm packages using cache
+        uses: bahmutov/npm-install@v1
+        with:
+          install-command: yarn --frozen-lockfile --network-concurrency 1
+      - name: Run Lint and Tests
+        run: yarn test
```

Comment:
It's kind of nice to break these out into individual commands so that it is clear which aspect of linting/testing failed in the CI build report.

Reply:
Won't it lint twice then?